Cargo.toml

```diff
@@ -68,6 +68,7 @@ alloy-primitives = { version = "0.8.12", default-features = false }
 # Maili
 maili-provider = { version = "0.1.0", default-features = false }
 maili-protocol = { version = "0.1.0", default-features = false }
+maili = { version = "0.1.0", default-features = false, features = ["rpc-types-engine"] }
```

Comment:
I think we can use [`maili-rpc-types-engine`](https://crates.io/crates/maili-rpc-types-engine)